images/win/scripts/Installers/Windows2016/Install-SSDT.ps1

```diff
@@ -6,7 +6,7 @@
 Import-Module -Name ImageHelpers -Force
 
 #SSDT for Visual Studio 2017 (15.9.3)
-$InstallerURI = 'https://download.microsoft.com/download/5/2/D/52DEF429-5B17-470C-82D9-654116080B15/SSDT-Setup-ENU.exe'
+$InstallerURI = 'https://download.microsoft.com/download/e/c/f/ecfccf97-b32c-4c31-b2fc-bff558635096/SSDT-Setup-ENU.exe'
```

Comment:
I guess we can parse download link from https://docs.microsoft.com/en-us/sql/ssdt/download-sql-server-data-tools-ssdt?view=sql-server-ver15#ssdt-for-vs-2017-standalone-installer and install the very latest version every time instead of hardcoding the link